lib/SRV/Volume/Base.pm

```diff
@@ -145,6 +146,18 @@ sub run {
         download_url => $self->download_url,
         type => $self->_type;
 
+    if ( $updater->in_progress ) {
```

Comment:
What was happening before this block was added?

Reply:
Was it just not checking at all if the request was already in progress?

Reply:
It looks like it was checking progress in `_background` (and still is), but that only got called if there was a `callback` parameter. Where is `$self->progress_filepath` defined? What parameters does that rely on?

Reply:
Right --- image bundling (especially for image/tiff) especially needs the extra check because making the zip archive takes a substantial amount of time. 
progress_filepath gets defined in `Base::_fill_params`, and is built off the marker (around #428)